pkg/bssTypes/types.go

```diff
@@ -56,7 +58,7 @@ type CloudInit struct {
 // provide a "default" selection which provides a way to supply default
 // parameters for any node which is not explicitly configured.
 type BootParams struct {
-	Hosts     []string  `json:"hosts,omitempty"`
+	Hosts     []string  `json:"hosts,omitempty"` // This list of hosts must be xnames
```

Comment:
(No change requested here, just commenting for posterity)

Unfortunately, it looks like there may have been premature optimization here as this field (according to the comment above) is supposed to support xnames _and_ "special names" (i.e. groups). However, the current BSS does not support boot groups with Postgres (yet). This may change with a solution to #50. However for now, this is fine as-is.